app/views/exchanges/manage_sep_types/_display_by_market.html.erb

```diff
@@ -1,15 +1,45 @@
-<div id="<%= market_kind %>" class="container tab-pane <%= display %>"><br>
-  <h3><%= l10n("exchange.manage_sep_types.titles")%></h3>
-  <p><%= l10n("exchange.manage_sep_types.sort_description")%></p>
-  <div class="container">
-    <div class="row">
-      <div data-controller="drag" data-drag-url="/exchanges/manage_sep_types/:id/move">
-        <% @sortable.where(market_kind: market_kind).active_by_state.each_with_index do |sort, i|%>
-            <div id='<%= "#{sort.reason}_#{sort.market_kind}" %>' data-id="<%= sort.id %>" data-ordinal_position="<%= sort.ordinal_position %>" data-index="<%=i%>" data-market_kind="<%=sort.market_kind%>" class="card card-body mb-4">
-              <%= sort.title%>
-            </div>
-        <%end%>
+<% if @bs4 %>
+  <div id="<%= market_kind %>" class="<%= display %>" data-controller="sep-types-list" data-target="sep-types-list.marketTab">
+    <% is_commonality_threshold_enabled = EnrollRegistry.feature_enabled?(:qle_commonality_threshold) %>
+    <% qles = @sortable.by_market_kind(market_kind).active_by_state %>
+    <% common_number = qles.common.length %>
+    <% threshold_index = common_number - 1 %>
```

Comment:
Could these vars maybe be set in a helper method in the controller? Or be put into a presenter class?

Reply:
I do agree, I think the `qles` var could be moved to the controller, especially now since I refactored the operation to pull QLEs from the DB instead of the submitted sort data. I think we defer that to the upcoming admin PR though